src/_pytest/outcomes.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, msg: Optional[str] = None, pytrace: bool = True) -> None:
         self.pytrace = pytrace
 
     def __repr__(self) -> str:
-        if self.msg:
+        if self.msg is not None:
```

Comment:
This changes the behavior of `str(Skipped(""))`. But if an empty string is explicitly passed I think it makes sense that the `str()` is empty.